css-layout-classes/styles.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-one-third {
+  display: flex;
+  flex-direction: column;
```

Comment:
You can remove these, generally column classes only call for a size to be set on them, the `row` class is usually the flex container.